src/aasthaJJun20/IntegerPairSum.java

```diff
@@ -0,0 +1,19 @@
+package aasthaJJun20;
+
+public class IntegerPairSum {
+	void findPairs(int[] input, int sum){
+		System.out.println("Integer numbers, whose sum is equal to value: "+sum);
+		for(int index=0; index<input.length; index++){
+			for(int innerIndex = 0; innerIndex < input.length; innerIndex++){
+				if(sum == input[index]+input[innerIndex])
```

Comment:
This is correct but to safer side always do () for mathematics operation in java